src/main/java/com/pubnub/api/endpoints/presence/SetState.java

```diff
@@ -131,4 +155,18 @@ protected boolean isAuthRequired() {
         return true;
     }
 
+    @Nullable
+    private String getStateParamValue() throws PubNubException {
```

Comment:
Maybe it's just me, but when I see `getXXX` I expect a getter, which usually doesn't do much work

Reply:
Ok. I can rename it to composeStateParamValue